docs/spec/spec.md

```diff
@@ -1980,6 +1983,21 @@ service on new graphql:Listener(9090) {
 ```
 >**Note:** It is recommended to disable introspection in production environments until it is required.
 
+#### 10.1.8 Constraint Configurations
+
+The `validation` field is used to enable or disable the validation of constraints defined on GraphQL input types. If constraint validation support is enabled, the GraphQL service verifies all constraints set on the GraphQL inputs when executing the resolver. By default, constraint validation is enabled for Ballerina GraphQL services.
+
+###### Example: Disable Constraint Validation Support
+
+```ballerina
+@graphql:ServiceConfig {
+    validation: false
```

Comment:
We should need a more expressive name here. Maybe `validateConstraints`? What is the name used in HTTP?

Reply:
Used `validation` to be consistent with other std-libs. HTTP and other std-libs use the `validation`.